#543 causes a typescript issue when building with the latest version of `rc-pagination`

The change in #543 is causing a typescript issue when I am attempting to build with the latest version of `rc-pagination`
I'm seeing:
```
> tsc -b

node_modules/rc-pagination/lib/index.d.ts:2:1 - error TS1383: Only named exports may use 'export type'.

2 export type * from './interface';
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

If you change this to `export * from `./interface` I believe things will do what you want and avoids this typescript error. This works fine because `interface.ts` is just a collection of types

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,2 +1,2 @@
 export { default } from './Pagination';
-export type * from './interface';
+export type { PaginationLocale, PaginationProps } from './interface';

